# Testing

## Bugs
### 1
PS C:\Users\CJ\Milestone-3--The-Snail-Hunt> python run.py
 * Serving Flask app 'snailmanager'
 * Debug mode: on
An attempt was made to access a socket in a way forbidden by its access permissions
PS C:\Users\CJ\Milestone-3--The-Snail-Hunt> 

#### Fix
